lib/LWP/UserAgent.pm

```diff
@@ -1988,6 +2023,47 @@ is in the object's C<requests_redirectable> list,
 false if the proposed redirection is to a C<file://...>
 URL, and true otherwise.
 
+=head1 BEST PRACTICES
+
+The default settings can get you up and running quickly, but there are settings
+you can change in order to make your life easier.
+
+=item cookie_jar
```

Comment:
Looks like you're using `=item` without a `=over` section here, but you do have a `=back` later on. Would it make more sense to `=head2` these sections so we could have direct URLs for them in metacpan and whatnot later to point people to?

Reply:
There already is a `=head2 cookie_jar`, so that might not work here. I'm open to some other way of doing this, though.

Reply:
🤔 and no pod checker that was complaining ?

Reply:
There was a failing test that Travis caught.  I just didn't run my tests locally.  :(

Reply:
Might I propose a different name to not conflict with `cookie_jar`? Since this is a Best Practices thing, it can be named anything we like. Maybe `=head2 Handling Cookies` or some such?